apply_extra.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+
+
+bsdtar -xf writersolo.deb
+tar -xvf data.tar.xz
```

Comment:
This makes the output show up on terminal while installing change it to:

```
bsdtar -Oxf writersolo.deb 'data.tar.xz'|bsdtar -xf - --exclude='usr'
```

Reply:
I removed the 2nd part, but forgot at the time to remove the -v flag to not show verbose output. I changed it later, if you're reading this now but haven't scrolled down yet.